Support for new domains

wojtekk opened the pull request:
Changes:

- support for new domains
- obsolete elements removed

Assumptions for new domains:

- the site will be available under 3 domains:
  - `www.whatisneeded.pl`
  - `www.shchopotribno.pl`
  - `www.copotrzebne.pl`
- domains without www redirect to domains with www
- decision about language should be made by frontend
- decision about redirecting to one domain i.e. `www.copotrzebne.pl` could be made by the frontend
- header `Host` will be passed to the backend and in the future could be used to decide about language, redirect, ...

wojtekk committed:
feat: Remove obsolete elements

wojtekk committed:
chore: Move tflint --init to make init script

wojtekk committed:
feat: Add english and ukrainian domain

sjedlikowski commented:
(not exactly related to this PR) Do we need `environments/prod/outputs.tf`? Are those values going to be used anywhere? :thinking:

wojtekk commented:
Good question, thank you - my intention was to show which information are used out of infra

- `remote_state_bucket` - used in terraform to define bucket
- `remote_state_replica_bucket` - could be used to restore state during downtime
- `s3_deployer_access_key_id` - used to deploy front
- `s3_deployer_access_key_secret` - used to deploy front
- `s3_deployer_bucket` - used to deploy front, just added
- `zone_name_servers` - NS used to delegate domain
- `zone_name_servers_en` - NS used to delegate domain
- `zone_name_servers_ua` - NS used to delegate domain

Removed:

- `s3_deployer_user_name`
- `zone_id`
- `zone_name`
- `zone_id_en`
- `zone_name_en`
- `zone_id_ua`
- `zone_name_ua`

wojtekk committed:
refactor: Cleanup output variables

wojtekk commented:
accepted on slack: https://copotrzebnepl.slack.com/archives/C036SRZV658/p1650020381286529?thread_ts=1649972883.514119&cid=C036SRZV658
